libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -2474,6 +2474,54 @@ string YulUtilFunctions::copyArrayFromStorageToMemoryFunction(ArrayType const& _
 		}
 	});
 }
+string YulUtilFunctions::stringConcatFunction(vector<Type const*> const& _argumentTypes)
```

Comment:
I don't think we need a new function on the yul level, since it is identical to bytes concat.